airflow/jobs.py

```diff
@@ -1513,6 +1518,16 @@ def _log_file_processing_stats(self,
                        if processor_start_time else None)
             last_run = processor_manager.get_last_finish_time(file_path)
 
+            file_name = file_path[len(dags_folder) + 1:]
+            dag_name = os.path.splitext(file_name)[0].replace(os.sep, '.')
```

Comment:
A dag file could contain multiple DAGs - is there a reason to not use dag_id here?